old-addon/README.md

```diff
@@ -0,0 +1,209 @@
+[👈 Back to README](../README.md)
```

Comment:
As I said earlier, all this content should be moved into the main readme. This readme should only contain information about why the old addon is still here and how to use it instead of the new one.